get-platform

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+
+"""
+platform-lite: A lightweight, extensible Python script for platform detection.
+
+Supported flags:
+  --os                  : Print the OS name (linux, darwin, etc.)
```

Comment:
```suggestion
  --os                  : Print the OS name (linux, macos, etc.)
```
Because `get_os()` replaces `darwin` by `macos`, right?

Reply:
correct, I will fix that